sdk/formrecognizer/azure-ai-formrecognizer/tests/test_custom_forms_from_url.py

```diff
@@ -173,7 +179,9 @@ def test_form_multipage_labeled(self, client, container_sas_url, blob_sas_url):
         forms = poller.result()
 
         for form in forms:
-            self.assertEqual(form.form_type, "form-"+model.model_id)
+            self.assertEqual(form.form_type, "custom:"+model.model_id)
+            self.assertEqual(form.form_type_confidence, 1.0)
+            self.assertIsNotNone(form.model_id)
```

Comment:
I wonder if this types of verifications could be added into a function that all similar test can call. So whenever we add new functionality we don't forget to validate this again